ci/do_ci.ps1

```diff
@@ -0,0 +1,20 @@
+$ErrorActionPreference = "Stop";
+trap { $host.SetShouldExit(1) }
+
+. "$PSScriptRoot\build_setup.ps1"
+Write-Host "building using $env:NUM_CPUS CPUs"
+
+function bazel_debug_binary_build() {
+  echo "Building..."
+  pushd "$env:ENVOY_SRCDIR"
+    bazel  $env:BAZEL_BASE_OPTIONS.Split(" ") build $env:BAZEL_BUILD_OPTIONS.Split(" ") -c dbg "//source/exe:envoy-static"
+    $exit = $LASTEXITCODE
```

Comment:
I sadly don't even know how to review PowerShell; are there any @envoyproxy/maintainers who do?